### PR TITLE
[MIM-1968] Mim 1968 adjust expansion box

### PR DESCRIPTION
**Pull request by ssb-cgn**
[Link to ticket: MIM-1968](https://statistics-norway.atlassian.net/browse/MIM-1968)

### CODE REVIEW

File: src/main/resources/assets/styles/_article.scss [hunk 1]
```diff
@@ -56,23 +56,31 @@
 
   .part-fact-box {
     padding: 0;
+    margin: 16px 0;
```
Comment (johnnadeluy) [inline src/main/resources/assets/styles/_article.scss:59]: Should we use rem here instead? 16px = 1rem

File: src/main/resources/assets/styles/_article.scss [hunk 2]
```diff
@@ -89,27 +97,28 @@
     }
   }
 
-  blockquote  {
+  blockquote {
     padding-left: 40px !important;
```
Comment (johnnadeluy) [inline src/main/resources/assets/styles/_article.scss:101]: Use rem here as well?

File: src/main/resources/assets/styles/_article.scss (hunk 1 above)
Comment (ssb-cgn) [reply src/main/resources/assets/styles/_article.scss]: Fixed

File: src/main/resources/assets/styles/_staticVisualization.scss [hunk 3]
```diff
@@ -1,29 +1,18 @@
 .part-static-visualization {
   .footnote {
-      list-style-type: none;
-      margin-top: 40px;
-      padding-left: 0;
+    list-style-type: none;
+    margin-top: 40px;
```
[the remark this answers is not in the document]
Comment (ssb-cgn) [reply src/main/resources/assets/styles/_staticVisualization.scss]: Fixed

**Review by johnnadeluy — APPROVED** [review]
Good job🎖

**Pull request merged**